package.json

```diff
@@ -3,19 +3,22 @@
   "version": "2.1.0",
   "description": "Simple in-memory vinyl file store",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "prepublish": "tsc"
   },
   "repository": "SBoudrias/mem-fs",
   "author": "Simon Boudrias",
   "license": "MIT",
-  "files": [
-    "index.js"
-  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "dependencies": {
+    "typescript": "^4.2.4",
```

Comment:
Typescript also should be a `devDependencies` no?

Reply:
:facepalm: fixed, sorry